webapp/components/profile_popover/profile_popover.jsx

```diff
@@ -243,16 +260,31 @@ export default class ProfilePopover extends React.Component {
                     </a>
                 </div>
             );
-            dataContent.push(webrtc);
         }
 
+        return '';
+    }
+
+    render() {
+        const popoverProps = Object.assign({}, this.props);
```

Comment:
Ah, I didn't see those getting passed in anywhere. Thanks for doing that though since it'll be